titus-server-master/src/main/java/com/netflix/titus/master/eviction/service/quota/job/TaskRelocationLimitController.java

```diff
@@ -79,7 +94,31 @@ public long getQuota() {
             }
         }
 
-        return quota;
+        return quota > 0
+                ? quotaBuilder.withQuota(quota).withMessage("Per task limit is %s", perTaskLimit).build()
+                : quotaBuilder.withQuota(0).withMessage("All tasks of the job reached its maximum eviction limit %s", perTaskLimit).build();
```

Comment:
I think this message needs to be rephrased.

Reply:
Will try